dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
+    <template #title>New Dataset - GDV</template>
+    <template #content>
+      <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+        <p class="font-medium text-xl">Loading GDV data...</p>
+        <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
+      </div>
+      <div v-else class="grid uploadFormWrapper">
+        <div id="uploadForm" class="text-left uploadForm col-9">
+          <FormKit
+            v-model="companyAssociatedGdvData"
+            :actions="false"
+            type="form"
+            :id="formId"
+            :name="formId"
+            @submit="postGdvData"
```

Comment:
Definitely better than the current warning

Reply:
I just investigated how I can paste a custom valdiation message here.
It is pretty easy to do it in general:

See this example from the FormKit docs:

![image](https://github.com/d-fine/Dataland/assets/99255296/26ba24e2-3e83-4adf-adeb-9e33f4cbd1ff)


However because the UploadConfig.ts is autogenerated, I cannot simply type a custom validation message into it.
In order to have custom validation messages, I'd need to extend the devtools to be able to handle this and write this into the UploadConfig.
I feel like that is not worth the effort for now. Especially because the upload config generation and its structure itself will be re-worked by Marc in the next weeks and months.

Please close this if that is ok with you.

Reply:
Will be investigated at the end of the ticket.